### PR TITLE
fix(clapi) Import fails on password type macros

### PR DESCRIPTION
**Pull request by callapa**
**Fixes**
Importing may result in an error when the type of a password macro is null in the file to be imported.
Avoids the successive addition of single quote characters

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 19.10.x
- [ ] 20.04.x
- [x] 20.10.x
- [x] 21.04.x (master)

## Checklist

- [x] I have followed the **coding style guidelines** provided by Centreon
- [x] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [x] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Review by Nohzoh — DISMISSED** [review]

**Review by sc979 — DISMISSED** [review]

File: www/install/createTables.sql [hunk 1]
```diff
@@ -1672,7 +1672,7 @@ CREATE TABLE `on_demand_macro_host` (
   `host_macro_id` int(11) NOT NULL AUTO_INCREMENT,
   `host_macro_name` varchar(255) NOT NULL,
   `host_macro_value` varchar(4096) NOT NULL,
-  `is_password` tinyint(2) DEFAULT NULL,
+  `is_password` tinyint(2) DEFAULT 0,
```
Comment (sc979) [inline www/install/createTables.sql:1675]: kindly add the upgrade script

**Review by sc979 — CHANGES_REQUESTED** [review]

File: www/install/createTables.sql (hunk 1 above)
Comment (callapa) [reply www/install/createTables.sql:1675]: indeed

**Review by jeremyjaouen — DISMISSED** [review]

**Review by loiclau — DISMISSED** [review]

File: www/install/php/Update-21.04.0-beta.1.php [hunk 2]
```diff
@@ -18,3 +18,54 @@
  * For more information : contact@centreon.com
  *
  */
+
+// error specific content
+$versionOfTheUpgrade = 'UPGRADE - 21.04.0-beta.1 : ';
```
Comment (sc979) [inline www/install/php/Update-21.04.0-beta.1.php:23]: shouldn't it be a 20.10.x script ?

Comment (sc979) [reply www/install/php/Update-21.04.0-beta.1.php:23]: will you create new PRs for the 19.10.x and 20.04.x versions ?

**Review by sc979 — CHANGES_REQUESTED** [review]

File: www/install/php/Update-21.04.0-beta.1.php (hunk 2 above)
Comment (callapa) [reply www/install/php/Update-21.04.0-beta.1.php:23]: Yes of course

Comment (sc979) [reply www/install/php/Update-21.04.0-beta.1.php]: I removed these versions in the description

**Review by sc979 — APPROVED** [review]
@callapa, I restored an empty 21.04.0-beta.1.php upgrade script.
This shouldn't block the cherry-pick on the 20.10.x branch

**Review by jeremyjaouen — APPROVED** [review]

**Pull request merged**